packages/common/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * as schemas from './llm/schemas'
```

Comment:
**chore:** that's not what I meant;

should be
```
common.openai.generateContent
common.openai.schemas
common.sentry // for sentry stuff later
```

Reply:
- `schemas` doesn't belong to OpenAI, as they represent the generic input/output that any LLM integration can use.
- Don't love the use of `common` as a forceful namespace (we can still do that in the consumer by just importing `* as common`).

Reply:
- sorry you're right! Then `llm.schemas` or `llmSchemas` would be better
- sure that's what I meant `import * as common`

Reply:
Done in https://github.com/botpress/botpress/pull/13134/commits/420e88527fdee7ef2c50191c4ffa150b9d89643e